src/main/scala/au/csiro/variantspark/cli/ImportanceCmd.scala

```diff
@@ -104,7 +104,16 @@ class ImportanceCmd extends ArgsApp with SparkApp
 
   @Option(name="-sr", required=false, usage="Random seed to use (def=<random>)", aliases=Array("--seed"))
   val randomSeed: Long = defRng.nextLong
-   
+  
```

Comment:
The current convention is to start all the random forest related options with `-rxxx` and `--rf-xxx` , e.g `-rmt` and `--rf-mtry`.
So there should be renamed to `-rmd` and `--rf-max-depth` etc.
Also, please move them up above `-sr` so that they are together with other random forest options.

(Nice to have/consider):
Refactoring all of the random forest options out of `ImportanceCmd` to `args.RandomForestArgs` in the same way as for instance `ImportanceOutputArgs`

Reply:
Updated the code to reflect both of these changes.